opentelemetry-api/tests/events/test_event.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+
+from opentelemetry._events import Event
+
+
+class TestEvent(unittest.TestCase):
+    def test_event(self):
+        event = Event("example", 123, attributes={"key": "value"})
+        self.assertEqual(event.name, "example")
+        self.assertEqual(event.timestamp, 123)
+        self.assertEqual(event.attributes, {"key": "value"})
```

Comment:
Event name [manifests as an attribute with key `event.name`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-sdk.md). How is that reflected here?

Reply:
@soumyadeepm04 

From the [api spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md#event-data-model):

> All Events have a [event.name attribute](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/events.md),

Reply:
I don't really think I understand completely what I need to change here 😅. Could you explain this a bit more?

Reply:
By the api definition, an `Event` needs to have an attribute with key `event.name` and value of the `Event` 's `name` field.

Reply:
@soumyadeepm04 care to handle this? So in this test it should be asserted that there is an event.name attribute and we should also add a test that in case of `event.name` passed in attributes by the user the one from the Event have precedence.

Reply:
https://github.com/open-telemetry/opentelemetry-python/pull/4054#pullrequestreview-2251274376